spec/index.bs

```diff
@@ -8040,19 +8040,21 @@ dictionary GPURenderPipelineDescriptor
                 1. Let |layout| be a new [$default pipeline layout$] for |pipeline| if
                     |descriptor|.{{GPUPipelineDescriptorBase/layout}} is {{GPUAutoLayoutMode/"auto"}},
                     and |descriptor|.{{GPUPipelineDescriptorBase/layout}} otherwise.
-                1. If any of the following conditions are unsatisfied:
-                    [$generate a validation error$], [$invalidate$] |pipeline|, and stop.
+                1. All of the requirements in the following steps |must| be met.
```

Comment:
Why is `must` wrapped in |pipes| like a variable here, and below?

Reply:
I should say what I said offline here too: it's to give us a way to highlight which items in the algorithm are requirements. Maybe not the best style possible but it lets us mix steps with requirements clearly.